compiler/qsc_partial_eval/src/management.rs

```diff
@@ -133,7 +133,7 @@ impl Backend for QuantumIntrinsicsChecker {
     ) -> Option<std::result::Result<Value, String>> {
         match name {
             "BeginEstimateCaching" => Some(Ok(Value::Bool(true))),
-            "EndEstimateCaching" | "GlobalPhase" => Some(Ok(Value::unit())),
+            "EndEstimateCaching" | "GlobalPhase" | "ConfigurePauliNoise" => Some(Ok(Value::unit())),
```

Comment:
looks like `ApplyIdleNoise` is missing from this one.

Reply:
I think the comment mentioning "functions" is a bit misleading, because `GlobalPhase` is also an operation and like `ApplyIdleNoise` is a no-op for codegen:
https://github.com/microsoft/qsharp/blob/5ac52cd3e105dbdecc26fe93966683f878974b57/library/std/src/Std/InternalHelpers.qs#L67-L69
They both fall into a special category of quantum operations that are simulator specific and don't need to be generated into QIR, so I think they should both end up here.

Reply:
Yes, this makes sense! I added it here AND in lib.rs to be ignored for partial evaluation. That's what was missing.